code/game/machinery/computer/communications.dm

```diff
@@ -94,10 +100,19 @@
 		// Login function.
 		var/list/access = usr.get_access()
 		if(allowed(usr))
-			authenticated = COMM_AUTHENTICATION_HEAD
+			authenticated = COMM_AUTHENTICATION_CAPT
 		if(ACCESS_CAPTAIN in access)
 			authenticated = COMM_AUTHENTICATION_CAPT
+		if(ACCESS_CENT_COMMANDER in access)
+			if(!is_admin(ui.user))
```

Comment:
Any reason you've done the whole `ui.user` thing instead of just `usr`?

Reply:
usr is less reliable than getting the direct user, isnt it?

Reply:
Not heard of that when relating with TGUI. Its been used on every other UI (and even this one in the next line), just wondered on why the approach.

Reply:
I can change them to usr if thats requested, but I try to stay away from usr when possible, since inherent byond variables can be unpredictable at times